web-admin/src/features/dashboards/share/ShareDashboardButton.svelte

```diff
@@ -29,29 +19,18 @@
   }
 </script>
 
-<Popover class="relative">
-  <PopoverButton use={[popperRef]}>
+<DropdownMenu>
+  <DropdownMenuTrigger>
     <Button type="secondary">Share</Button>
```

Comment:
Good point. Instead of adding builders to our `Button` component, I used our `buttonClasses()` function to apply the `secondary` button classes to the `DropdownMenuTrigger` itself.